### PR TITLE
Clear lastVersion in a case of returning an empty TokenBucketState

### PR DESCRIPTION
**Pull request by oskarwojciski**
Solving #29

### CODE REVIEW

File: tokenbucket.go [hunk 1]
```diff
@@ -364,6 +364,12 @@ func (t *TokenBucketRedis) State(ctx context.Context) (TokenBucketState, error)
 	var values []interface{}
 	var err error
 	done := make(chan struct{}, 1)
+
+	if t.raceCheck {
+		// reset in a case of returning an empty TokenBucketState
+		t.lastVersion = 0
+	}
```
Comment (mennanov) [inline tokenbucket.go:371]: Do you think it makes sense to add the test that covers this?

Comment (oskarwojciski) [reply tokenbucket.go:371]: @mennanov I can commit the test mentioned here: https://github.com/mennanov/limiters/issues/29. It is not so "generic" compared to other tests but the issue is not related to other backends.

Comment (mennanov) [reply tokenbucket.go:371]: Hm.. let's keep the tests generic.
I will merge this PR as is.

**Review by mennanov — APPROVED** [review]

**Comment by mennanov** [comment]
Thanks for contributing!

**Pull request merged**